### DISCUSSION TITLE
Support letter signatory highlighter is not entirely a bad idea...

### DISCUSSION THREAD

**Discussion by ycpei** [rms-support-letter/rms-support-letter.github.io#5632] [General]
I'd be interested in knowing or working with some of my fellow signatories.

Also on the grand scheme of things I feel like it would be a lost opportunity to have 5000+ people rallying behind one cause and then dissipate. Shall we form a new organisation?

**Comment by Bluesboy** [comment]
They just want to intimidate people. Despite that their threats doesn't mean a shit.

**Reply by waltercool** [reply]
I just hope those companies and organizations who supported the coup receive some monetary punishment from users.

While I may agree some individuals can have strong opinions pro and cons, organizations shouldn't never take any side here... specially supporting something vile as removing the head of FSF and all their committee members

**Comment by pda0** [comment]
I think we should definitely form something like a community and not lose sight of each other. Molly de Blanc is now like a wounded beast, she will never calm down and surrender.
Also, it would be great if more people knew about what happened. Their approach only works while maintaining the appearance of unanimous condemnation. If a large number of people stand up and ask "what the heck?", the power of fear crumbles like a house of cards.

**Reply by ycpei** [reply]
We'd be doing everyone a favour if we end cancel culture once and for all.

**Comment by 6r1d** [comment]
I think promoting things like [critical thinking](https://en.wikibooks.org/wiki/A-level_Critical_Thinking) and scientific method generally is due. I may sound nerdy, but scientific method is about **collecting actual evidence**, assessing it properly and working with data. It's not what modern journalism is about. It's not what modern cancel culture is about. We may make a little site with infographics to help this cause and not lose sight of each other, yes.

The fact of "acting collectively" should be gradually addressed and we should improve our methods.

While many are concerned about free software, just think about progress of humanity. Current culture is obsessed with things **wasting our time**. And we should strive to actually improve.

**Reply by ludovitkramar** [reply]
The modern culture of "liberate" and "fight against" really should end, actually building something is what needs to be done.

**Reply by jobbautista9** [reply]
The word "liberate" and phrase "fight against" has been bastardized by the elite. We need to change that. And this defense we are doing for RMS is a good start.
